optimoveapi/api/actions.py

```diff
@@ -1,25 +1,37 @@
+import logging
+
 from ..utils import date_to_str
 
 
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.WARNING)
+
+
 class Actions(object):
     """Namespace class for action-related endpoints"""
 
     def __init__(self, transport):
         self._transport = transport
 
     def get_executed_campaign_details(self, date):
+        logger.debug('Getting executed campaign for {}'.format(date))
```

Comment:
You should format strings by using `logger.info('123 %s 789', '456')`. Benefit of that is when you mess in string template, logger line won't crash.